conf/drupal/config/config_ignore.settings.yml

```diff
@@ -2,4 +2,5 @@ _core:
   default_config_hash: UVH1aJ4b44UM-VdPVN7hNNuuVqfReJxwfVeDQH1Hvsk
 ignored_config_entities:
   - mass_utility.settings
+  - mass_analytics.settings
```

Comment:
If your goal with ignoring is env specific overrides, then I suggest adding code to https://github.com/massgov/openmass/blob/develop/docroot/sites/default/settings.acquia.php#L100-L131 instead.

Reply:
I don't think it's environment specific.

Reply:
As Joe mentioned in our discussion about this ticket, we don't want the URL to be publicly available, which will the case if we export it.

Reply:
Aha. What we have done for this in the past is just put a placeholder URL in the config yml file and then do a config override in settings.php like https://github.com/massgov/openmass/blob/develop/docroot/sites/default/settings.php#L137-L139

Reply:
With the current implementation of analytics we don't have anything like that. It is configured from the UI with route_iframes module, And I don't see a benefit for complicating this to that level as you mentioned was done in the past. We have mass_utility settings ignored with config_ignore for example. I don't see a benefit for changing approach for this requirement.

Reply:
I see my suggestion as simpler, not more complicated. Getting rid of config ignore is a simplification. And I dont see how the config ignore approach will work anywhere but Prod. The value will get reset on every --db-refresh. Thats inconvenient on local dev, for example. You will have to get an URL from Prod every time you want to work on this feature.

Feel free to move this to In Progress and assign to me and I am happy to complete the ticket.